### PR TITLE
Removed references to Power of Us Hub

### PR DESCRIPTION
**Pull request by sam-knox**
Updated labels, objects, and other components to remove references to the Power of Us Hub.

# Critical Changes

# Changes
 - We removed references and links to the Power of Us Hub and replaced them with links to the Trailblazer community.

# Issues Closed

# Community Ideas Delivered

# Features Intended for Future Release

# Features for Elevate Customers

# New Metadata

# Deleted Metadata

### CODE REVIEW

**Review by lparrott — APPROVED** [review]

File: force-app/main/default/classes/UTIL_HtmlOutput_CTRL.cls [hunk 1]
```diff
@@ -53,7 +53,7 @@ public with sharing class UTIL_HtmlOutput_CTRL {
 
     /** @description The map of allowed urls and their temporary substitution values */
     private static final Map<String, String> SUBSTITUTION_BY_ALLOWED_URL = new Map<String, String> {
-        '<a href="https://powerofus.force.com/' => '|hubURL|',
```
[the remark this answers is not in the document]
Comment (sam-knox) [reply force-app/main/default/classes/UTIL_HtmlOutput_CTRL.cls]: Ah, ok. Sorry about that!

**Review by dfuller114 — APPROVED** [review]
thanks Sam!

File: README.md [hunk 2]
```diff
@@ -2,20 +2,16 @@
...
+* <a href="https://trailhead.salesforce.com/trailblazer-community/groups/0F94S000000kHitSAE?tab=discussion&sort=LAST_MODIFIED_DATE_DESC" target="_blank">Ask questions or get help</a>
+* <a href="https://github.com/SalesforceFoundation/NPSP/issues/new" target="_blank">Log a confirmed Issue</a> or <a href="https://ideas.salesforce.com/s/search#t=All&sort=relevancy" target="_blank">Feature Request</a>
 * <a href="https://powerofus.force.com/NPSP_Documentation" target="_blank">User Documentation</a>
-* Check out existing <a href="https://github.com/SalesforceFoundation/NPSP/labels/bug" target="_blank">bugs</a> and <a href="https://powerofus.force.com/hub-ideas?product=NPSP" target="_blank">feature and enhancement requests.</a>
+* Check out existing <a href="https://github.com/SalesforceFoundation/NPSP/labels/bug" target="_blank">bugs</a> and <a href="https://ideas.salesforce.com/s/search#t=All&sort=relevancy&f:@sfcategoryfull=[Nonprofit%7CNonprofit%20Success%20Pack]" target="_blank">feature and enhancement requests.</a>
```
Comment (jesslopez-sf) [inline README.md:8]: @sam-knox  - and same with this mention of 'bugs' -- maybe link to Known Issues instead?
https://trailblazer.salesforce.com/issues_index?tag=SFDO%20Nonprofit%20Success%20Pack

Comment (sam-knox) [reply README.md:8]: @jesslopez-sf : Yeah we can update that. The switch to Known Issues happened (or we knew of it) well after we made these initial updates to remove the Hub references. I can ask the devs about this, but being that it's near the end of the development cycle I'll wait until after feature freeze.

I imagine there's similar updates needed in other repos too.

**Pull request merged**